ERCS/erc-7769.md

```diff
@@ -160,6 +159,22 @@ Response:
 }
 ```
 
+##### Support for [EIP-7702](./eip-7702.md) authorizations
+
+On networks with [EIP-7702](./eip-7702) activated, the `UserOperation` object may also contain an `eip7702Auth` tuple.
+When providing the `eip7702Auth` tuple, the fields `factory` and `factoryData` have a modified behaviour.
```

Comment:
No, they are unrelated. 
You can have 0x7702 factory, without specifying a new tuple. This makes sure that the UserOp will be included only if the account want modified.
The reverse is also possible, but less recommended.

Reply:
Right. Rephrased this paragraph.